README.md

```diff
@@ -183,26 +174,26 @@ with open(dst, "wb") as f:
     f.write(env.file.save())
 ```
 
-### [Asset](UnityPy/files/SerializedFile.py)
+### Asset
 
-Assets are a container that contains multiple objects.
+Assets \([SerializedFile class](UnityPy/files/SerializedFile.py)\) are a container that contains multiple objects.
 One of these objects can be an AssetBundle, which contains a file path for some of the objects in the same asset.
 
 All objects can be found in the `.objects` dict - `{ID : object}`.
 
 The objects with a file path can be found in the `.container` dict - `{path : object}`.
 
-### [Object](UnityPy/files/ObjectReader.py)
+### Object
 
-Objects contain the _actual_ files, e.g., textures, text files, meshes, settings, ...
+Objects \([ObjectReader class](UnityPy/files/ObjectReader.py)\) contain the _actual_ files, e.g., textures, text files, meshes, settings, ...
 
 To acquire the actual data of an object it has to be read first. This happens via the `.read()` function. This isn't done automatically to save time because only a small part of the objects are of interest. Serialized objects can be set with raw data using `.set_raw_data(data)` or modified with `.save()` function, if supported.
 
 ## Important Object Types
 
-All object types can be found in [UnityPy/classes](UnityPy/classes/).
+Now UnityPy uses auto generated classes while all object types can also be found in [UnityPy/classes/legacy_patch/](UnityPy/classes/legacy_patch/).
 
-### [Texture2D](UnityPy/classes/Texture2D.py)
+### Texture2D
```

Comment:
Wouldn't it be better to correct the link, to make it point to the `classes/legacy_patch/` folder, instead of removing it?

Reply:
Yep, that sounds like a good idea to point them to the `.pyi` files in `legacy_patch`.
The only objects of these, that don't have their own hand-written implementation, are `TextAsset`, `MonoBehaviour` and `Font`. I guess they can be left without a link then, unless you think it's a good idea to link to `generated.py` for these

Reply:
Long term I plan to move all legacy patches into separate, optional plugins, that act like a wrapper around the specific classes instead of extending them.

So just linking to ``generated.py`` is fine.


The existence of ``legacy_patch`` and that it extends some classes would make sense tho.

Reply:
With the current state of the documentation, I think it is important to mention `legacy_patch`, because otherwise people might not even discover it, or know that is being used (due to its name).